skiko/src/commonTest/kotlin/org/jetbrains/skia/util/assertCloseEnough.kt

```diff
@@ -48,6 +52,25 @@ private inline fun FontMetrics.isCloseEnoughTo(b: FontMetrics, epsilon: Float) =
         strikeoutThickness.isCloseEnoughTo(b.strikeoutThickness, epsilon) &&
         strikeoutPosition.isCloseEnoughTo(b.strikeoutPosition, epsilon)
 
+private inline fun LineMetrics.isCloseEnoughTo(b: LineMetrics, epsilon: Float): Boolean =
+    startIndex == b.startIndex &&
+    endIndex == b.endIndex &&
+    endExcludingWhitespaces == b.endExcludingWhitespaces &&
+    endIncludingNewline == b.endIncludingNewline &&
+    isHardBreak == b.isHardBreak &&
+    ascent.isCloseEnoughTo(b.ascent, epsilon) &&
+    descent.isCloseEnoughTo(b.descent, epsilon) &&
+    unscaledAscent.isCloseEnoughTo(b.unscaledAscent, epsilon) &&
+    height.isCloseEnoughTo(b.height, epsilon) &&
+    width.isCloseEnoughTo(b.width, epsilon) &&
+    baseline.isCloseEnoughTo(b.baseline, epsilon) &&
+    lineNumber == b.lineNumber
+
+
+
+
+
+
```

Comment:
```suggestion

```

Reply:
not sure if it created a correct patch. It's meant to keep only one new line

Reply:
yep, pushed